### ISSUE TITLE
Sprite examples

### ISSUE THREAD

**Issue by MorenoIgor** [alekmaul/pvcollib#5]
Hello there! I'm experimenting with the library and getting the hang of it, however the sprite examples don't seem to work. The release version has some naming typos for the assets, but even after correcting those, no sprites show on the emulator.

The game examples appear to deal with sprites differently, by writing directly to vram. Is this the intended way? Or maybe there is something I am missing?

Cheers!

**Comment by alekmaul** [comment]
Hello, it is strange as we tested all examples.
Could you please explain which example exactly (we have more than one sprite example).
Also, what do you mean with release version ? and which typos assets did you find ?
By the way, fixing typo will not help to have something on emulator.

**Comment by MorenoIgor** [comment]
Hey, thanks for the reply!

The release version I'm talking about is the latest, 1.5.1.

The examples in questions are graphics/sprites/simplesprite and animatedsprite. The first one (simplesprite) references 'TILpacsprite' and 'SZTILpacsprite' which lack the 'gfx' suffix added by the bitmap conversion tool. (This appears to have been fixed on the main branch already).

However, after fixing these and building, no sprites show up on emulators (tried both blueMSX and ColEm). I've also tried copying the contents of the generated .inc file directly to the C source file, as I thought maybe the sprite data was not properly defined, but that didn't work either.

I tried looking into other examples to see if I could figure it out, but both Bunny and Diamond appear to use custom assembly routines and not the library functions such as sprite_set, etc.

I've attached the zipped example folder - the only thing I changed from the original was include 'gfx' for the assets. The built ROM file is also in there, so you may take a look.

[simplesprite.zip](https://github.com/alekmaul/pvcollib/files/11303166/simplesprite.zip)

**Comment by alekmaul** [comment]
sorry, i thought it was about pvsneslib and not pvcollib ...
ok, I will check the example ^^

**Comment by MorenoIgor** [comment]
Hi! Dropping by to check if there's any answer to my problem, hehe. I really look forward to using this lib for some projects. :D

**Comment by alekmaul** [comment]
Yeah, I was really busy with lots of work on other projectsbut, you know, i'm using the lib in all my projects. SO, even if one example is not working for you, I think it's a good idea to try a project and test the lib ;)

**Comment by MorenoIgor** [comment]
Sure thing! Just wanted to know the intended way of using the functions, but I'll fiddle around a bit. Thanks! =)
